### PR TITLE
OSDOCS-6951 Adding pd-balanced disktype to GCP  install

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-6951

Link to docs preview:
https://63183--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installation-config-parameters-gcp#installation-configuration-parameters-additional-gcp_installation-config-parameters-gcp

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@rna-afk @marcbrown2024 Hi Aditya and Mark, could one of you take a look at this docs PR and the corresponding [release note PR](https://github.com/openshift/openshift-docs/pull/63184)? Thank you

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -1442,7 +1442,7 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
 |`platform.gcp.defaultMachinePlatform.osDisk.diskType`
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].
-|Either the default `pd-ssd` or the `pd-standard` disk type. The control plane nodes must be the `pd-ssd` disk type. Compute nodes can be either type.
+|The default disk type for all machines. The control plane nodes must be the `pd-ssd` disk type. Compute nodes can be the `pd-ssd`, `pd-balanced`, or `pd-standard` disk types.
```
Comment (kowen-rh) [inline modules/installation-configuration-parameters.adoc:1445]: For consistency between sentences, consider removing "the" before "control plane nodes".

I'm assuming that compute nodes cannot have more than one disk type. Consider rewriting to something like the following:

```adoc
Compute nodes can be a `pd-ssd`, `pd-balanced`, or `pd-standard` disk type.
```

**Review by kowen-rh — COMMENTED** [review]
One nitpick, otherwise looks good to me!

/remove-label peer-review-in-progress
/label peer-review-done

File: modules/installation-configuration-parameters.adoc (hunk 1 above)
Comment (bscott-rh) [reply modules/installation-configuration-parameters.adoc:1445]: I went with "use":
```
Compute nodes can use the `pd-ssd`, `pd-balanced`, or `pd-standard` disk types.
```

**Pull request merged**